internal/engine/wazevo/backend/isa/arm64/lower_instr.go

```diff
@@ -1012,6 +1014,73 @@ func (m *machine) lowerIcmp(si *ssa.Instruction) {
 	m.insert(cset)
 }
 
+func (m *machine) lowerVIcmp(si *ssa.Instruction) {
+	x, y, c, lane := si.VIcmpData()
+	flag := condFlagFromSSAIntegerCmpCond(c)
+	arr := ssaLaneToArrangement(lane)
+
+	var ext extMode
+	if c.Signed() {
+		ext = extModeSignExtend64
+	} else {
+		ext = extModeZeroExtend64
+	}
```

Comment:
I don't believe this is needed. Could you give me the rationale

Reply:
it was originally there in `lowerIcmp()` for size == 64 (on which this was originally based; for a little, I tried to share the same function but it made little sense), but it is probably useless; tests still pass